test/sequential/test-http-pipeline-flood.js

```diff
@@ -53,18 +74,13 @@ function parent() {
     assert.equal(connections, 1);
     // The number of requests we end up processing before the outgoing
     // connection backs up and requires a drain is implementation-dependent.
```

Comment:
I think you can remove the comment in its entirety now, it's not hugely insightful or enlightening on its own.